wordpress-mobile/WordPress-Android
# Remove Comments view all screen

Fixes #9979 
The comments endpoint doesn't return correct data for the view all screen so we decided to remove it from both platforms. 

To test:
* Go to Stats/Insights/Comments block
* Notice that the "View more" button is gone

Update release notes:
